clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterRegistration.java

```diff
@@ -131,12 +131,20 @@ public void execute() {
           startTask(new RegisterSchemaTask(0));
         } else {
           Microservice newMicroservice = serviceCenterClient.getMicroserviceByServiceId(serviceResponse.getServiceId());
-          dealIsSwaggerDifferent(newMicroservice);
+          if (isSwaggerDifferent(newMicroservice)){
+            if (canRewriteSwagger()){
+              LOGGER.warn("Service has already registered, but schema ids not equal. However, it will continue to register");
+              startTask(new RegisterSchemaTask(0));
+            }else {
+              LOGGER.warn("Service has already registered, but schema ids not equal. However, it will continue to register. "
+                      + "If you want to eliminate this warning. you can Change the microservice version or delete the old " +
+                      "microservice info and try again. eg: version:1.0.0 -> 1.0.1");
```

Comment:
Service has already registered, but schema ids not equal. Ignore and continue to register.